### PR TITLE
Add Design Readme

### PR DESCRIPTION
**Pull request by rachelcope**
Reason for Change
===================
* Add Design README
* https://trello.com/c/LLWawjxt/212-contributor-process-for-design

Changes
=======
* Add onboarding resources and design process/workflow

### CODE REVIEW

File: DESIGN-README.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+## Design Files
+* Sketch file (request link)
+* [Icon Set](https://www.emojione.com/developers/download?key=8YNLFBYZAB47ST8ZDZV3)
+
+## Styleguide/Branding
+This application was designed using an Atomic design system.
+
+* [Styleguide](http://demo.getcalfresh.org/styleguide)
+* [Styleguide code](https://github.com/codeforamerica/gcf-backend/tree/master/app/views/styleguides)
```
Comment (jessieay) [inline DESIGN-README.md]: this is a private repo -- are we worried about linking to it from a public repo?

File: DESIGN-README.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+## Front-End Design Contribution
+Please read the [Contributing Guidelines](https://github.com/codeforamerica/michigan-benefits/blob/master/CONTRIBUTING.md) for details on the process for code contribution and submitting pull request.
+
+## Workflow
+* A design feature or change is initiated and discussed via a Trello card.
+* All new Trello cards are added to the "Design/UI Backlog" Trello list.
+* During a weekly planning meeting, prioritized Trello cards are added to the "Next Up" Trello list. Cards in this list should have enough detail to be actionable. Top priority items are at the top of the list.
+* When someone starts working on a task, they move the Trello card into "In Progress" and assign themselves so that others know they are working on it.
+* When finished, the designer should request a review by the product manager or another designer.
```
Comment (jessieay) [inline DESIGN-README.md]: what is the preferred way to request a review?

Comment (rachelcope) [reply DESIGN-README.md]: A review on Trello makes sense. I'll add that in

File: DESIGN-README.md (hunk 1 above)
Comment (rachelcope) [reply DESIGN-README.md]: @alanjosephwilliams Are we concerned about this? Should i remove the link to the private repo?

Comment (jessieay) [reply DESIGN-README.md]: per alan in standup, this is NOT a problem

**Pull request merged**